index.html

```diff
@@ -999,6 +999,10 @@ <h2 id="BaseAudioContext">
               suspended.
             </p>
             <p>
+              Let <em>pendingResumePromises</em> be a list of promises that
```

Comment:
This needs to be initialized to an empty, ordered list in the `AudioContext` ctor so that the prose is a bit more stronger.